application/org.openjdk.jmc.ui.common/src/main/java/org/openjdk/jmc/ui/common/util/ThemeUtils.java

```diff
@@ -0,0 +1,30 @@
+package org.openjdk.jmc.ui.common.util;
+
+import org.eclipse.jface.preference.JFacePreferences;
+import org.eclipse.jface.resource.ColorRegistry;
+import org.eclipse.swt.graphics.RGB;
+import org.eclipse.ui.PlatformUI;
+
+public class ThemeUtils {
```

Comment:
**suggestion:** I believe the themes can change at runtime. `isDarkMode` will compute this value always. But to avoid the computation on each call, you could store a variable here and update this value each time the theme changes.

Something like:

```java
public class ThemeUtils {
  private static boolean isCurrentThemeDark = isDarkMode();

  static {
    PlatformUI.getWorkbench().getThemeManager().addPropertyChangeListener(() -> isCurrentThemeDark = isDarkMode())
  }

  public static boolean isDarkThemeTheme() { return isCurrentThemeDark; }
}
```

This way the value is updated anytime the theme changes. I'm not quite sure when to register the listener, I put it in static block as an example, but maybe this is not recommended in the Eclipse RCP programing idioms.

Note I kept the original name of `isDarkMode`, but feel free to rename to anything meaningful in your implementation.